doc/source/examples/extended_examples/index.rst

```diff
@@ -25,6 +25,8 @@ with other programs, libraries, and features in development.
 +------------------------------------------------------+--------------------------------------------------------------------------------------------+
 | :ref:`hpc_ml_ga_example`                             |  Demonstrates how to use PyMAPDL in a high-performance computing system managed by SLURM.  |
 +------------------------------------------------------+--------------------------------------------------------------------------------------------+
+| :ref:`stochastic_fem_example`                        |  Demonstrates using PyMAPDL for stochastic FEA via the Monte Carlo simulation.             |
```

Comment:
You might want to replace via with the word with. See https://developers.google.com/style/word-list#via.